tests/numpy/unop_test.py

```diff
@@ -29,7 +29,10 @@ def test_not():
     B = np.zeros((5, 5), dtype=np.int64).astype(np.bool_)
     regression = np.logical_not(A)
     nottest(A, B)
-    assert np.alltrue(B == regression)
+    if np.lib.NumpyVersion(np.__version__) >= '2.0.0b1':
```

Comment:
should we change this to 2.0.0?

Reply:
I don't think it makes a practical difference, but we should change it for consistency.